### ISSUE TITLE
dist-fx shouldn't transpile for..of statements (and some other es6 features)

### ISSUE THREAD

**Issue by n1k0** [Kinto/kinto.js#320]
Firefox supports them out of the box.

**Comment by leplatrem** [comment]
Nice!

That was the only ES6 feature transpiled. The other one (commonjs) is required by browserify.

By the way, we're thinking replacing `Object.assign()` with spread (see #304).
```js
d = {...r, foo: "bar"}
```
instead of:
```js
d = Object.assign({}, r, {foo: "bar"})
```

**Comment by n1k0** [comment]
First step (for..of) landed in #321.

**Comment by mozmark** [comment]
As @leplatrem says, this was probably the only one we can deal with. I think moving the fx specifics in-tree is the next thing but that's a separate bug.
